src/regions/ca/info/ca-qc/reporting.md

```diff
@@ -10,7 +10,7 @@ The [Self-Care Guide](https://publications.msss.gouv.qc.ca/msss/en/document-0024
 
 If you are worried about COVID‑19, you can call 418-644-4545, 514-644-4545, 450-644-4545, 819-644-4545 and 1-877-644-4545 (toll free) elsewhere in Québec.
 
-If you have symptoms such as a cough or cough that gets worse, fever, difficulty breathing or sudden loss of sense of smell without nasal congestion with or without loss of taste:
+If you have symptoms such as new cough or cough that gets worse, fever, difficulty breathing or sudden loss of sense of smell without nasal congestion with or without loss of taste:
```

Comment:
Is this from the government site as well? The wording is a bit weird.

`If you have symptoms such as a new cough, a cough that has worsened, fever, difficulty breathing or sudden loss of sense of smell without nasal congestion with or without loss of taste:`

Reply:
Oui c'est copy/paste du gouvernement... C'est QC ... ils ne parlent pas bien anglais lol. Le QC fait toujours ça, changer un petit mot par jour pour me tenir occupée -_-'